packages/mobile/lib/routers/routers/dashboard/profile-router.ts

```diff
@@ -29,6 +29,8 @@ export class ProfileRouter extends Subrouter<ProfileRoute> {
                     nextRoute = ProfileRoute.Settings
                 } else if (networkStatus) {
                     nextRoute = ProfileRoute.NetworkStatus
+                } else if (backup) {
+                    nextRoute = ProfileRoute.Backup
```

Comment:
I would hook the rest of the logic to enter the password to export (we can use the electron logic and replace it with the native sharesheet later on).
Upon export success we have to make sure the router goes back to the profile action view, and the backup local storage is updated accordingly to display the button again or not

Reply:
I moved the export result handling with the notification into the router and added a previous() call, when the export succeeds.